package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "putasset",
-  "version": "4.0.1",
+  "version": "4.0.2",
```

Comment:
No point to change `patch` I'll change `minor` because it's a new feature 😉

Reply:
You are right